### ISSUE TITLE
Add issue forms for the GitHub Issues

### ISSUE THREAD

**Issue by LtDan33** [pyscript/pyscript#260]
# What is the idea?

Add the new "Issue Forms" beta GitHub feature to this repo for bugs and feature requests.

# Why is this needed

There are a lot of issues that are being created, and its hard to parse through them all when they are in many different formats. There is also no guidance when anyone is creating a new feature request or bug, and so its hard to know what should be added in order to get something addressed.

**Comment by LtDan33** [comment]
(Note that I can't assignee anyone right now but I'm actively working on this)

**Comment by antocuni** [comment]
@LtDan33 have a look at #252 and #255 .
Feel free to just continue/improve #252

**Comment by LtDan33** [comment]
@antocuni Thanks for the heads up :) I didn't look hard enough! The two forms/templates are actually separate file types so I think it makes more sense to have a separate PR, as the entire file would be replaced. #255 makes sense for me to use.
